### ISSUE TITLE
Consider removing basic SHOULD language re HTTP responses

### ISSUE THREAD

**Issue by xmlgrrl** [KantaraInitiative/wg-uma#312]
In the [Grant Sec 3](https://docs.kantarainitiative.org/uma/ed/oauth-uma-grant-2.0-04.html#protocol-flow-details-sec) intro and [FedAuthz Sec 1.5](https://docs.kantarainitiative.org/uma/ed/oauth-uma-federated-authz-2.0-04.html#rfc.section.1.5), @jricher suggests:

Is this a meaningful SHOULD? Seems like the obvious and default thing to do in an HTTP protocol.

The language in question:

Grant: "The recipient of each request message SHOULD respond unless it detects a security concern, such as a suspected denial of service attack that can be mitigated by rate limiting."

FedAuthz: "The authorization server SHOULD respond to each request from the resource server unless it detects a security concern, such as a suspected denial of service attack that can be mitigated by rate limiting."

**Comment by xmlgrrl** [comment]
Options:

1. Leave the language in.
1. Remove both paragraphs.
1. (Some other option I can't think of...?)

Discussion:

This language, which originally appeared in Core and I only recently copied from the new Grant to the new FedAuthz, was mostly there to give some sort of normative force to our wholly declarative/descriptive language around the UMA request-response messaging. E.g., "The client does X" and then "The authorization server does Y in response", rather than "The client MUST do X" and then "The authorization server MUST (or SHOULD or MAY) do Y in response".

I believe that "not using normative language" was one of Justin's early critiques of the UMA spec, but we found it really hard to use normative language in the actual flow in a natural way; we tried it and ended up taking it out.

Note that OAuth and OIDC both use declarative/descriptive language when it comes to messaging flows.

If a response is always at the discretion of the server-side recipient based on context, we could drop with no harm. Apparently the other similar specs felt no need to include such a warning.
